src/pages/examples/styling-clusters.hbs

```diff
@@ -11,6 +11,9 @@ layout: example.hbs
 <!-- Load Clustered Feature Layer from CDN -->
 <script src="https://unpkg.com/esri-leaflet-cluster@2.0.0/dist/esri-leaflet-cluster.js"></script>
 
+<!-- Load Esri Leaflet Renderers -->
+<script src="https://unpkg.com/esri-leaflet-renderers@2.0.6"></script>
```

Comment:
How are you using `esri-leaflet-renderers` plugin? I'm not noticing a difference in the sample if I remove it.

Reply:
Yeah, please remove it.  I'm not actually using it. Good catch!